grunt/nodemon.json

```diff
@@ -0,0 +1,5 @@
+{
+  "server": {
+    "script": "bin/index.js"
```

Comment:
Not sure what the utility is here. My thinking goes along the lines that this is like an example file, so shouldn't be included by default. It's a reminder to include it if there is a `bin` script in the material repo, no?